guides/pro/encoders.md

```diff
@@ -77,9 +76,17 @@ You can combine several encoders into a single chain of versioned encoders. Pass
 
 ```ruby
 # Define some encoders ...
-NewSecureEncoder       = GraphQL::Pro::Encoder.define { ... }
-OldSecureEncoder       = GraphQL::Pro::Encoder.define { ... }
-LegacyInsecureEncoder  = GraphQL::Pro::Encoder.define { ... }
+class NewSecureEncoder < GraphQL::Pro::Encoder
+  # ...
+end
+
+class OldSecureEncoder < GraphQL::Pro::Encoder
+  # ...
+end
+
+class LegacyInsecureEncoder < GraphQL::Pro::Encoder
+  # ...
+end
 
 # Then order them by priority:
 VersionedEncoder = GraphQL::Pro::Encoder.versioned(
```

Comment:
@rmosolgo is this still how versioned encoders are defined?

I don't recall how we started defining this and we've diverged a little bit into our own encoder class

Reply:
Yup, it's the same :+1: